grakn-graph/src/main/java/ai/grakn/graph/internal/GraknComputerImpl.java

```diff
@@ -50,17 +51,18 @@
  */
 public class GraknComputerImpl implements GraknComputer {
     private final Graph graph;
-    private final Class<? extends GraphComputer> graphComputer;
+    private final Class<? extends GraphComputer> graphComputerClass;
```

Comment:
Looks like the class name is passed at construction time, but the actual computer is lazily created only when needed. Not sure if there is a penalty to create the computer right away.